citestwheel.Dockerfile

```
@@ -51,12 +53,14 @@ ENV PATH="/pyenv/versions/${PYTHON_VER}/bin/:$PATH"
 # Install the AWS CLI
 # Needed to download wheels for running tests
 # Install the AWS CLI
-RUN mkdir -p /aws_install && cd /aws_install && \
-    curl "https://s3.amazonaws.com/aws-cli/awscli-bundle.zip" -o "awscli-bundle.zip" && \
-    unzip awscli-bundle.zip && \
-    ./awscli-bundle/install -i /usr/local/aws -b /usr/local/bin/aws && \
-    cd / && \
-    rm -rf /aws_install
+RUN <<EOF
+mkdir -p /aws_install && cd /aws_install
```

```suggestion
mkdir -p /aws_install
cd /aws_install
```
Let's remove the `&&` since we're no longer using it anywhere else.